src/features/duck-player.js

```diff
@@ -46,6 +71,20 @@ export default class DuckPlayerFeature extends ContentFeature {
             this._messaging = new Messaging(context, config)
             return this._messaging
         }
+        if (this.platform.name === 'macos') {
+            const context = new MessagingContext({
```

Comment:
Can you construct this outside the if statement and share it with the Windows code above?